src/notebook-jobs-panel.tsx

```diff
@@ -62,7 +62,13 @@ export class NotebookJobsPanel extends VDomRenderer<JobsModel> {
     this.node.setAttribute('aria-label', trans.__('Notebook Jobs'));
   }
 
-  showListView(view: JobsView.ListJobs | JobsView.ListJobDefinitions): void {
+  showListView(
+    view: JobsView.ListJobs | JobsView.ListJobDefinitions,
+    newlyCreatedId?: string,
+    newlyCreatedName?: string
+  ): void {
+    this.model.listJobsModel.newlyCreatedId = newlyCreatedId;
+    this.model.listJobsModel.newlyCreatedName = newlyCreatedName;
```

Comment:
Wait, but binding these attributes to the list jobs model persists them. That means even when I refresh the page, don't these attributes remain bound and thus the banner persists?

I had meant for these attributes to be stored as plain old properties on the NotebookJobsPanel class (outside of `this.model`), which are not persisted.